### PR TITLE
feat: support configurable server side ssl ca check

### PR DESCRIPTION
**Pull request by earayu**
## Checklist

-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required

### CODE REVIEW

**Comment by newborn22** [comment]
Where should the CA be placed if the user want to use SSL/TLS? Maybe it will be better if the path of CA is configurable too?

**Comment by earayu** [comment]
> Where should the CA be placed if the user want to use SSL/TLS? Maybe it will be better if the path of CA is configurable too?

It is already configurable.
```
mysql_server_ssl_ca=
mysql_server_ssl_cert=
mysql_server_ssl_key=
```

File: go/vt/servenv/servenv.go [hunk 1]
```diff
@@ -414,6 +415,14 @@ func init() {
 		OnParseFor(cmd, stats.RegisterFlags)
 	}
 
+	for _, cmd := range []string{
+		"vtctld",
+		"vtgate",
```
Comment (newborn22) [inline go/vt/servenv/servenv.go:420]: It seems that only vtgate needs to and have the ability to validate certs?

**Pull request merged**